### ISSUE TITLE
install problem for R ver 3.30

### ISSUE THREAD

**Issue by long-jian** [dmlc/mxnet#2357]
Hi, I have an install problem that when I followed the steps to install mxnet, and this Rstudio showed this error comment:

Warning in install.packages :
  package mxnet is not available (for R version 3.3.0)

**Comment by thirdwing** [comment]
Which OS are you using?

**Comment by phunterlau** [comment]
if it is related to `dmlc.ml` domain expiration, please try again.

**Comment by long-jian** [comment]
My OS is Whindows10, and I set my R and Rstudio at `C:\` not `C:\Program Files`.  Is this way cause my situation? 
I fixed this problem last night, my solution is get the `.zip` of `mxnet`, then install it at local path by Rstudio. Before install `mxnet` there is a package which is `DiagrammeR` seems depended on mxnet, and also not available for R version 3.3.0. I install it through github. Then I successfully install `mxnet` at my computer.

>library(devtools)
>devtools::install_github('rich-iannone/DiagrammeR')

mxnet zip:
https://github.com/dmlc/drat/tree/gh-pages/bin/windows/contrib/3.2#sthash.JdeQgtZ1.dpuf

Thanks for your replying, I would like to help anyone has same problem with me. But I am not sure is it a good way or not.
